script/lint.js

```diff
@@ -99,20 +100,26 @@ const LINTERS = [{
   test: filename => filename.endsWith('.js') || filename.endsWith('.ts'),
   run: async (opts, filenames) => {
     const eslint = new ESLint({
-      cache: true,
+      // Do not use the lint cache on CI builds
+      cache: !process.env.CI,
+      cacheLocation: `node_modules/.eslintcache.${crypto.createHash('md5').update(fs.readFileSync(__filename)).digest('hex')}`,
       extensions: ['.js', '.ts'],
       fix: opts.fix
     });
     const formatter = await eslint.loadFormatter();
     let successCount = 0;
+    const results = [];
     for (const result of await eslint.lintFiles(filenames)) {
       successCount += result.errorCount === 0 ? 1 : 0;
-      if (result.errorCount !== 0 || result.warningCount !== 0) {
-        console.log(formatter.format([result]));
-      } else if (opts.verbose) {
+      results.push(result);
+      if (opts.verbose && result.errorCount === 0 && result.warningCount === 0) {
```

Comment:
I'm not sure this change is needed to fix anything. AFAIK `formatter.format()` is only creating the output for human consumption (what you see with the CLI). If there are no warnings or errors, the output (for the default formatter) is blank. Bumping it out to doing all results at once means that it won't be interleaved with the "no errors or warnings" in verbose mode.

Reply:
Calling `formetter.format` on each individual file results in really hard to results when multiple files have warnings / errors.  This way is consistent with the old output we got by running `eslint` directly.